src/main/resources/com/google/api/codegen/java/initcode.snip

```diff
@@ -1,7 +1,17 @@
 @extends "java/common.snip"
 
 @snippet initCode(initCodeSpec)
-  @join line : initCodeSpec.lines
+  @if initCodeSpec.argDefaultLines
+    // should be commented to not clobber args
```

Comment:
Modify the templates for all languages so that this comment (and the closing one) have a `FIXME` prefix so we remember to address this.

(FYI, in the `.snip`s, I'm leaving `TODO` as something the sample user needs to do, and `FIXME` for things we have to do to get this feature out the door.)

Reply:
done